uniflow-core/src/main/kotlin/io/uniflow/core/flow/data/UIData.kt

```diff
@@ -0,0 +1,3 @@
+package io.uniflow.core.flow.data
+
+interface UIData
```

Comment:
Could this be a sealed class to unlock the powers of exhaustive `when` expressions?

Reply:
it's also interesting to allow to use generic State and not force compeltely people to have a complete When ... best is to find a tradeof